spec/functional/resource/group_spec.rb

```diff
@@ -267,14 +267,14 @@ def create_group
     end
 
     describe "when removing members" do
-      it "raises an error for a non well-formed domain name" do
+      it "does not raise an error for a non well-formed domain name" do
```

Comment:
Is this behavior change desired?  I understand about not raising an error for a non-existant member, but why not raise an error if the member name has a space in it?